README.md

```diff
@@ -99,8 +99,8 @@ To run the project, execute the following command:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- **Project 1: add Person, Student, and Teacher classes.**
-- **Project 2: use the "Decorator" design pattern.**
+- [x] Project 1: add Person, Student, and Teacher classes.
+- [x] Project 2: use the "Decorator" design pattern.
 - **Project 3: set up associations.**
 - **Project 4: add basic UI.**
 - **Project 5: refactor your code.**
```

Comment:
- [ ] As requested by the previous reviewer, kindly fix the broken link to your `MIT` license agreement
![image](https://github.com/elad237/OOP-school-library/assets/81932478/50de6034-df42-41c0-9340-e3faae64f3c3)

It doesn't go to the license when a user clicks on it
![image](https://github.com/elad237/OOP-school-library/assets/81932478/00db5f1c-54ac-4aec-9ac3-62898d9171c4)